selfservice/flow/login/organizations.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2025 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package login
+
+import (
+	"net/http"
+
+	"github.com/gofrs/uuid"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+
+	"github.com/ory/kratos/identity"
+	"github.com/ory/kratos/session"
+)
+
+var organizationFilter = []StrategyFilter{func(s Strategy) bool {
+	a, b := s.(interface {
+		SupportsOrganizations() bool
```

Comment:
Let the strategy tell us if it support organizations or not, rather than inferring it.